src/components/Logs.tsx

```diff
@@ -87,25 +90,33 @@ function Logs({
             editorRef.current = editor;
         },
         reset: () => {
+            setNetworkFailure(false);
             setEmptyResponses(0);
         },
     };
 
     useInterval(
         async () => {
-            const { data: viewLogsResponse } = await fetchLogs();
-
-            if (viewLogsResponse && viewLogsResponse.length > 0) {
-                const logsReduced = viewLogsResponse.map((logData) => {
-                    return generateLogLine(logData);
-                });
-                setOffset(offset + viewLogsResponse.length);
-                setLogs(logs.concat(logsReduced));
-                if (disableIntervalFetching) {
+            const fetchLogsResponse = await fetchLogs();
+
+            if (fetchLogsResponse.error) {
+                setNetworkFailure(true);
+                setEmptyResponses(MAX_EMPTY_CALLS + 1);
```

Comment:
If there was an error it is probably due to a networking issue. So we should just stop trying to fetch logs.